### PR TITLE
docs: update action/config page

### PR DESCRIPTION
**Pull request by sergkudinov**

### CODE REVIEW

File: docs/content/action/config.md [hunk 1]
```diff
@@ -3,72 +3,67 @@ title: Config
...
+// Register the action
+.registry.register('execute', ({config, metadata}) => {
+  // Check the argument
+  if (metadata.argument != null)
+    config.command = metadata.argument
   // More code goes here
-});
-.execute('whoami', function(err, {stdout}){
-  console.info('I am ' + stdout.trim());
 })
```
Comment (wdavidw) [inline docs/content/action/config.md:40]: I think using `execute` and reimplementing it is confusing. Do you agree or shall we leave it like this ?

Comment (sergkudinov) [reply docs/content/action/config.md:40]: agree

File: docs/content/action/config.md [hunk 2]
```diff
@@ -3,72 +3,67 @@ title: Config
 sort: 1
 ---
 
-# Metadata "config"
+# Configuration properties
 
-Config are used to contextualise the handler function.
+Configuration properties are used to contextualize the handler function.
```
Comment (wdavidw) [inline docs/content/action/config.md:8]: What is the difference between a metadata and an config, config are properties tied an action while metadata influence the behavior of multiple actions, we shall see this. You description is to generic

File: docs/content/action/config.md [hunk 3]
```diff
@@ -3,72 +3,67 @@ title: Config
...
+.registry.register('my_action', ({config, metadata}) => {
+  // Check the argument
+  if (metadata.argument != null)
+    config.command = metadata.argument
   // More code goes here
-});
-.execute('whoami', function(err, {stdout}){
-  console.info('I am ' + stdout.trim());
 })
 ```
```
Comment (wdavidw) [inline docs/content/action/config.md:41]: no reference to the `argument_to_config` plugin ?

File: docs/content/action/config.md [hunk 4]
```diff
@@ -3,72 +3,67 @@ title: Config
...
 
-## Merging
+## Merging properties
 
-When multiple configs are passed, they will be merged with the last keys taking precedence over previously defined keys:
+When multiple configuration properties are passed, they are merged with the last keys taking precedence over previously defined keys. The example below asserts this behavior:
```
Comment (wdavidw) [inline docs/content/action/config.md:45]: Convert 1st sentence in 2 sentences

File: docs/content/action/config.md [hunk 5]
```diff
@@ -3,72 +3,67 @@ title: Config
...
+  .call({key: 'old value'}, {key: 'new value'}, ({config}) => {
+    assert.equal(config.key, 'new value')
+  })
+})()
 ```
 
-## Global definition
-
-Config passed to the Nikita session on instantiation are available globally to every handlers.
+Values set as `undefined` are passed but they will not overwrite previously defined properties:
```
Comment (wdavidw) [inline docs/content/action/config.md:58]: A value `undefined` will not..

**Review by wdavidw — CHANGES_REQUESTED** [review]

**Review by wdavidw — APPROVED** [review]

**Pull request merged**